### ISSUE TITLE
Problem with exporting duik rigging using IK

### ISSUE THREAD

**Issue by ovungLotha** [airbnb/lottie-web#1188]
I have created a simple animation using duik rigging. It shows up fine when i preview it on the web but when i preview it on android and iOS the hands and legs i rigged dont move, and it only shows those controller points moving.

**Comment by bodymovin** [comment]
Hi, Android and iOS players don't support expressions

**Comment by master96** [comment]
any solution for this ?

**Comment by master96** [comment]
@bodymovin

**Comment by bodymovin** [comment]
@master96 you can bake the expressions into keyframes

**Comment by master96** [comment]
@bodymovin  Yeah I did that it worked but JSON size spiked really high(it was 100 kb before now its 1.7 MB ). I think i'll have to wait till both android and ios start supporting expressions
